Source/Helper.swift

```diff
@@ -15,14 +15,13 @@ struct Helper {
       return CGAffineTransform.identity
     }
   }
-
+  
   static func videoOrientation() -> AVCaptureVideoOrientation {
-    switch UIDevice.current.orientation {
-    case .portrait: return .portrait
-    case .landscapeLeft: return .landscapeRight
-    case .landscapeRight: return .landscapeLeft
-    case .portraitUpsideDown: return .portraitUpsideDown
-    default: return .portrait
+    switch UIApplication.shared.statusBarOrientation {
```

Comment:
We want to get the orientation that the app is in before clicking camera and lock it into that.